composer.json

```diff
@@ -19,7 +19,7 @@
   ],
   "require": {
     "php": ">=7.0.13",
-    "illuminate/support": "^5.4 || ^6.0",
+    "laravel/framework": "^5.4|^5.8|^6.0|^7.0|^8.0|^9.0|^10.0",
```

Comment:
What's the reason for the switch from `illuminate/support` to `laravel/framework`?